### ISSUE TITLE
Leshrac's ultimate Pulse Nova ALWAYS require only 70 mana to activate, but always cost more than that (90/110/130, depending on Pulse Nova level)

### ISSUE THREAD

**Issue by xorox11** [jeffhill/Dota2#1326]
### Description

Example: If you activate a Level 2 Pulse Nova when you have 80 mana, you only spend 80 mana, but if you activate it when you have 110 mana, you spend 110 mana, in first case 30 mana goes to void.

**Comment by pseudoparenchymatous** [comment]
Check the notes here: https://dota2.fandom.com/wiki/Leshrac#Pulse_Nova

**Comment by xorox11** [comment]
> Check the notes here: https://dota2.fandom.com/wiki/Leshrac#Pulse_Nova

So just because wiki states how it works means its not a bug? mana cost shown of ability doesnt even matches the actual mana cost, its def a bug.

**Comment by Daar543** [comment]
> > Check the notes here: https://dota2.fandom.com/wiki/Leshrac#Pulse_Nova
> 
> So just because wiki states how it works means its not a bug? mana cost shown of ability doesnt even matches the actual mana cost, its def a bug.

Yeah, even if it was intended, the shown mana cost would scale with your current mana until reaching the cap.

**Comment by TheUnholyWrath** [comment]
Voodoo Restoration and Arctic Burn don't work like this. You should only have to pay 70 Mana to activate Pulse Nova but the Mana per Second (20/40/60) is also being applied on the first tick. Mana per Second should only be applying the second after you turn Pulse Nova on. Every time you turn it on you're paying for 1 second of Pulse Nova that you don't get

**Comment by TheUnholyWrath** [comment]
This was fixed in 7.32
